lib/ansible/modules/crypto/openssl_csr.py

```diff
@@ -221,6 +233,11 @@
     returned: changed or success
     type: list
     sample: [ 'clientAuth' ]
+basicConstraints:
```

Comment:
this block should be removed

Reply:
I've now included it in `dump`

Reply:
Ah, I was looking in the wrong section, this is the return section, not the documentation one. :-)